component/appcat_apiserver.jsonnet

```diff
@@ -0,0 +1,139 @@
+local common = import 'common.libsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local com = import 'lib/commodore.libjsonnet';
+local params = inv.parameters.appcat;
+local apiserverParams = params.apiserver;
+
+local image = params.images.apiserver;
+local loadManifest(manifest) = std.parseJson(kap.yaml_load('appcat/manifests/' + image.tag + '/' + manifest));
```

Comment:
I'm a bit confused about this load function. I can't see any `appcat/manifests/` folder, so I'm a bit lost from where you get those files in the first place.

Reply:
Ah figured it out... it only appears after generation 👍